src/__tests__/jest/actions/authUser.test.js

```diff
@@ -0,0 +1,87 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import moxios from "moxios";
+import expect from "expect";
+
+import {
+  START_FETCH,
+  SIGNUP_SUCCESS,
+  SIGNUP_ERROR
+} from "../../../redux/action_types";
+
+import {
+  signup,
+  signupSuccess,
+  signupError
+} from "../../../redux/actions/authUser";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore({});
```

Comment:
@kimotho-njoki Add store globally

Reply:
I believe I said define store within `beforeEach()`... on line 20 you can have `let store` then within `beforeEach()` have `store = mockStore({})`.

Reply:
@kimotho-njoki I thought the const in global was cleaner